### DISCUSSION TITLE
Enable or Disable spamassassin per mail server

### DISCUSSION THREAD

**Discussion by chrism417** [postalserver/postal#1997] [Feature suggestions]
We need a way to enable or disable spamassassin per mail server in our organization.

I haven't seen a way to do this so far in the Web UI, but if I missed it can this be added?

**Comment by willpower232** [comment]
Under Advanced Settings, you can see that outbound scanning is not enabled by default however incoming messages always will be. This means that messages you are trying to send will always be sent but any message someone else sends you might be held.

![image](https://user-images.githubusercontent.com/1619102/169508755-4f36da6d-b68f-4e7e-9cc6-719769ccc907.png)

If you are having problems receiving emails, you can adjust the incoming thresholds on the spam tab.

**Comment by chrism417** [comment]
yes you're correct, however, we have several domains we want to relay through postal. On one of these domains, we don't want spamassassin to run at all because there is such large volume going through the domain it will slow emails down, and we don't care about their spam score for these emails.
